packages/math/metaStable.ts

```diff
@@ -0,0 +1,45 @@
+import { OldBigNumber } from "@balancer-labs/sor";
```

Comment:
🤔 I'm a bit out of context here but how many BN implementations are we currently using?

Reply:
The @balancer/sor package uses a self-defined version (OldBigNumber) and one from ethersproject (BigNumber). I thought would be good to follow the same pattern here